modules/forms/src/components/field.tsx

```diff
@@ -244,6 +244,9 @@ export const InnerField = React.forwardRef((props: InnerFieldPropsInterface, ref
                                     value={ value }
                                     checked={ form.get(inputField.name) === value }
                                     onChange={ (event: React.ChangeEvent<HTMLInputElement>, { value }) => {
+                                        if(inputField?.onBefore && !inputField.onBefore(event,value)){
```

Comment:
Use ESLint for formatting

```suggestion
                                        if (inputField?.onBefore && !inputField.onBefore(event, value)) {
```